test/models/report_test.rb

```diff
@@ -3,7 +3,18 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'user can not edit report' do
+    report = reports(:first)
+    assert_not report.editable?(users(:bob))
+  end
+
+  test 'user can edit report' do
+    report = reports(:first)
+    assert report.editable?(users(:alice))
+  end
+
+  test 'Get the date when the daily report was created' do
+    report = reports(:second)
```

Comment:
```suggestion
    report = reports(:second)
    report.created_at = ...
```

こういう場合は例外的にcreated_atをセットするのもありです